src/Files.App/Views/Settings/TagsPage.xaml

```diff
@@ -341,7 +342,7 @@
 											<Button
 												x:Name="SaveChangesButton"
 												Click="CommitRenameTag_Click"
-												Content="{helpers:ResourceString Name=SaveChanges}"
+												Content="{helpers:ResourceString Name=Save}"
```

Comment:
Shouldn't the string resource `SaveChanges` also be removed?

Reply:
Whoops, you're right